app/packages/core/src/components/ColorModal/colorPalette/ColorPalette.tsx

```diff
@@ -131,7 +133,7 @@ const ColorPalette: React.FC<ColorPaletteProps> = ({
           </ColorSquare>
         ))}
         {colors.length < maxColors && (
-          <AddSquare onClick={handleColorAdd}>
+          <AddSquare onClick={handleColorAdd} data-cy="add-color">
```

Comment:
nit: prefix with `custom-colors-` + `add-color` in case of a future name collision and more context